lib/roby/interface/client.rb

```diff
@@ -273,6 +286,27 @@ def call(path, m, *args)
 
             # @api private
             #
+            # Asynchronously call a method on the interface or on one of the
+            # interface's subcommands
+            #
+            # @param [Array<String>] path path to the subcommand. Empty means on
+            #   the interface object itself.
+            # @param [Symbol] m command or action name. Actions are always
+            #   formatted as action_name!
+            # @param [Object] args the command or action arguments
+            def async_call(path, m, *args, &block)
+                if m.to_s =~ /(.*)!$/
+                    job_id = call(path, m, *args)
```

Comment:
Why do you keep the action calls synchronous ?

Reply:
My understanding is that action calls is non blocking, am I wrong? I am forwarding the call to the other (sync) call method to avoid copying the code that actually calls the action. Would you prefer if had `#start_job` here instead?

Reply:
It actually is blocking, it waits for the `job_id` that is returned by the remote side.

Reply:
I got that wrong then and as a result what I did is also wrong... Thanks for pointing that out

Reply:
> Thanks for pointing that out

:+1: Anytime. Thanks for what you're doing right now, it's really helpful.